monero-outreach-docs/translations/pt-br/storyboard/wallet_quickstart_pt-br.md

```diff
@@ -0,0 +1,57 @@
+
+Então, com tudo isso em mente, considere as seguintes opções: se você deseja rodar seu próprio nó completo, uma boa opção é a carteira GUI ou CLI oficial do Monero. A primeira oferece uma interface gráfica de usuário e a segunda, uma interface textual. Ambas podem ser configuradas tanto para rodar um nó completo quanto para se conectar com um nó remoto de uma maneira mais privada. Essas carteiras podem ser baixadas em [getmonero.org/downloads](https://web.getmonero.org/pt-br/downloads/). Para dispositivos Android, o Monerujo, em [monerujo.io](https://www.monerujo.io/), ou para iOS, a Cake Wallet, em [cakewallet.io](https://cakewallet.io/), são carteiras leves recomendadas do tipo que protegem sua chave de visualização. Se você preferir uma carteira leve do tipo que compartilha a chave de visualização, a carteira direto do navegador MyMonero, em [mymonero.com](https://mymonero.com/), é recomendada. A Edge Wallet, em [edge.app](https://edge.app/), é outra boa opção de carteira leve de compartilhamento de chave de visualização para ambos Android e iOS.
+
+Quando estiver escolhendo entre essas carteiras, tenha em mente que a maioria — exceto a Edge Wallet — tem um recurso extra em que elas podem ser configuradas para se conectar tanto com um servidor remoto de terceiros quanto com um servidor que você roda com um nó. Isso é ilustrado na tabela.
+
+## Carteiras oficiais
+
++ [getmonero.org/downloads](https://web.getmonero.org/pt-br/downloads/)
+Se você quer rodar um nó completo, uma boa escolha é a carteira GUI ou CLI oficial do Monero. A primeira oferece uma interface gráfica de usuário e a segunda, uma interface textual.
+
+## Carteiras móveis
+
++ [cakewallet.io](https://cakewallet.io/)
+Cake Wallet, uma carteira leve protetora da chave de visualização, está disponível para dispositivos iOS.
+
++ [monerujo.io](https://www.monerujo.io/)
+Monerujo, uma carteira leve protetora da chave de visualização, está disponível para dispositivos Android.
+
++ [edge.app](https://edge.app/)
+Edge Wallet está disponível para ambos Android e iOS. É uma carteira leve de compartilhamento de chave de visualização.
+
+## Carteiras web *(Otimizadas para conveniência)*
+
++ [mymonero.com](https://mymonero.com/)
+MyMonero é uma carteira leve direto do navegador. Está disponível também como app para iOs e como app de desktop.
+
+## Carteiras de hardware *(Otimizadas para segurança)*
+
++ [ledger.com](https://shop.ledger.com/?r=92d74dc2847a)
+Ledger é um dispositivo de altíssima segurança que armazena sua semente mnemônica.
+Não aplicável
```

Comment:
I'd like to check with both @erciccione and @lh1008 if line 48 should read "Not Applicable" or if it's just a mistake.

Reply:
This looks like a leftover from the draft, but @lh1008 will know better than me.

Reply:
hey, yes this has to be removed from the original English file. Let me fix it. Thank you for noticing it.

Reply:
Sorry guys its ok, leave the "not applicable". This is for the site https://www.monerooutreach.org/stories/monero_wallet_quickstart.php